modules/imaging_browser/templates/menu_imaging_browser.tpl

```diff
@@ -1,109 +1,109 @@
-    </div>
-    <div class="panel-body" id="panel-body">
-        <form method="post" action="main.php?test_name=imaging_browser">
-             <div class="row">
-                <div class="form-group col-sm-4">
-                    <label class="col-sm-12 col-md-4">
-                        {$form.pscid.label}
-                    </label>
-                    <div class="col-sm-12 col-md-8">
-                        {$form.pscid.html}
-                    </div>
-                </div>
-                 <div class="form-group col-sm-4">
-                    <label class="col-sm-12 col-md-4">
-                        {$form.DCCID.label}
-                    </label>
-                    <div class="col-sm-12 col-md-8">
-                        {$form.DCCID.html}
-                    </div>
-                </div>
-                 <div class="form-group col-sm-4">
-                    <label class="col-sm-12 col-md-4">
-                        {$form.VL.label}
-                    </label>
-                    <div class="col-sm-12 col-md-8">
-                        {$form.VL.html}
-                    </div>
+<div class="panel-heading" onclick="hideFilter();">
+    Selection Filter
+    <span class="glyphicon glyphicon-chevron-down pull-right" style="display:none" id="down"></span>
+    <span class="glyphicon glyphicon-chevron-up pull-right" id="up"></span>
+</div>
+<div class="panel-body" id="panel-body">
+    <form method="post" action="{$baseurl}/imaging_browser/">
+         <div class="row">
+            <div class="form-group col-sm-4">
+                <label class="col-sm-12 col-md-4">
+                    {$form.pscid.label}
+                </label>
+                <div class="col-sm-12 col-md-8">
+                    {$form.pscid.html}
                 </div>
             </div>
-            <div class="row">
-                <div class="form-group col-sm-4">
-                    <label class="col-sm-12 col-md-4">
-                        {$form.SiteID.label}
-                    </label>
-                    <div class="col-sm-12 col-md-8">
-                        {$form.SiteID.html}
-                    </div>
+             <div class="form-group col-sm-4">
+                <label class="col-sm-12 col-md-4">
+                    {$form.DCCID.label}
+                </label>
+                <div class="col-sm-12 col-md-8">
+                    {$form.DCCID.html}
                 </div>
-                <div class="form-group col-sm-4">
-                    <label class="col-sm-12 col-md-4">
-                        {$form.VisitQCStatus.label}
-                    </label>
-                    <div class="col-sm-12 col-md-8">
-                        {$form.VisitQCStatus.html}
-                    </div>
+            </div>
+             <div class="form-group col-sm-4">
+                <label class="col-sm-12 col-md-4">
+                    {$form.VL.label}
```

Comment:
It looks like it's not, it's just the diff.. the order in both the original and the new version is pscid, DCCID, VL, SiteID, VisitQCStatus.. for some reason GitHub shows the VisitQC and VL as switched even though they're not